### PR TITLE
Network Stats is now a gump

### PR DESCRIPTION
**Pull request by dissident76**
Double click to change amount of info displayed

### CODE REVIEW

File: src/Game/UI/Gumps/NetworkStatsGump.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                _label.Hue = NetClient.Socket.Statistics.RenderedText.Hue;
+                _label.Text = NetClient.Socket.Statistics.RenderedText.Text;
+            }
+            else
+            {
+                _label.Hue = 0x20;
+                _label.Text = "Disconnected";
+            }
+                       
+            if (!FullDisplayMode) _label.Text = _label.Text.Split('\n')[0];
```
Comment (andreakarasho) [inline src/Game/UI/Gumps/NetworkStatsGump.cs]: This is cause a lot of allocation in runtime. 
What is the purpose?

**Comment by andreakarasho** [comment]
Now i've a clear view of the PR.
If networkstatistics class does not draw text anymore, you can remove the RenderedText field and making the string into the gump directly

File: src/Game/UI/Gumps/NetworkStatsGump.cs (hunk 1 above)
Comment (dissident76) [reply src/Game/UI/Gumps/NetworkStatsGump.cs]: This is so the gump can be minimised with double click and show only ping value, without bytes in/out. So RenderedText gets trimmed right before the \n character that separates ping value and the rest. But can get rid of the minimised mode if it creates a lot of overhead

**Comment by dissident76** [comment]
Thought of moving the string creation to the gump class, but I wanted to change the existing NetStatistic class the least possible. If I move the string to the gump, I will have to create an Update method to calculate bytes in/out. But yeah, NetStatistics needs an overhaul -will give it another go tomorrow

**Pull request closed without merging**